src/main/groovy/edu/oregonstate/mist/invencycle/db/BikeDAO.groovy

```diff
@@ -11,6 +11,118 @@ import org.skife.jdbi.v2.sqlobject.Bind
+              VALUES (BIKE_FRAME_SIZE_SEQ.nextval, :frame_size_name, :frame_size_cm)
+               """)
+    void postFrameSize(@Bind("frame_size_name") String frame_size_name,
+                       @Bind("frame_size_cm") Integer frame_size_cm)
+    
+    @SqlUpdate("""
+              INSERT INTO SUSPENSION (SUSPENSION_ID, SUSPENSION_MAKE, SUSPENSION_MODEL, SUSPENSION_TRAVEL, SUSPENSION_TYPE)
+              VALUES (BIKE_SUSPENSION_ID_SEQ.nextval, :fork_make, :fork_model, :fork_travel, 'Fork')
+               """)
+    void postFork(@Bind("fork_make") String fork_make,
+                  @Bind("fork_model") String fork_model,
+                  @Bind("fork_travel") Integer fork_travel)
+
+    @SqlUpdate("""
+              INSERT INTO SUSPENSION (SUSPENSION_ID, SUSPENSION_MAKE, SUSPENSION_MODEL, SUSPENSION_TRAVEL, SUSPENSION_TYPE)
+              VALUES (BIKE_SUSPENSION_ID_SEQ.nextval, :shock_make, :shock_model, :shock_travel, 'Shock')
+               """)
+    void postShock(@Bind("shock_make") String shock_make,
+                  @Bind("shock_model") String shock_model,
+                  @Bind("shock_travel") Integer shock_travel)
+    
+    @SqlUpdate("""
+               INSERT INTO WHEEL (WHEEL_ID, WHEEL_SIZE, WHEEL_RIM_MAKE, WHEEL_RIM_MODEL, WHEEL_HUB_MAKE, WHEEL_HUB_MODEL, WHEEL_POSITION)
+               VALUES (BIKE_WHEEL_ID_SEQ.nextval, :wheel_size_front, :rim_make_front, :rim_model_front, :hub_make_front, :hub_model_front, 'Front')
+              """)
+    void postWheelFront(@Bind("wheel_size_front") Integer wheel_size_front,
+                        @Bind("rim_make_front") String rim_make_front,
+                        @Bind("rim_model_front") String rim_model_front,
+                        @Bind("hub_make_front") String hub_make_front,
+                        @Bind("hub_model_front") String hub_model_front)
+
+    @SqlUpdate("""
+               INSERT INTO WHEEL (WHEEL_ID, WHEEL_SIZE, WHEEL_RIM_MAKE, WHEEL_RIM_MODEL, WHEEL_HUB_MAKE, WHEEL_HUB_MODEL, WHEEL_POSITION)
+               VALUES (BIKE_WHEEL_ID_SEQ.nextval, :wheel_size_rear, :rim_make_rear, :rim_model_rear, :hub_make_rear, :hub_model_rear, 'Rear')
+              """)
+    void postWheelRear(@Bind("wheel_size_rear") Integer wheel_size_rear,
+                        @Bind("rim_make_rear") String rim_make_rear,
+                        @Bind("rim_model_rear") String rim_model_rear,
+                        @Bind("hub_make_rear") String hub_make_rear,
+                        @Bind("hub_model_rear") String hub_model_rear)
+
+    @SqlUpdate("""
+              INSERT INTO BIKE (BIKE_ID, BIKE_MAKE, BIKE_MODEL, BIKE_YEAR, BIKE_BIKE_TYPE_ID, BIKE_FRAME_SIZE_ID, BIKE_MSRP, BIKE_DERAILUER_FRONT_ID, BIKE_DERAILUER_REAR_ID, BIKE_FORK_ID, BIKE_SHOCK_ID, BIKE_WHEEL_FRONT_ID, BIKE_WHEEL_REAR_ID, BIKE_BRAKE_FRONT_ID, BIKE_BRAKE_REAR_ID, BIKE_CREATE_DATE)
+              VALUES (BIKE_ID_SEQ.nextval, :bike_make, :bike_model, :bike_year,
+
```

Comment:
this line is too long. I've updated the codeNarc rule to limit the max length per line of code, which is 100 characters per line. It will also make your code easier to read :)